python/lsst/ts/standardscripts/maintel/take_triplet_comcam.py

```diff
@@ -0,0 +1,244 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["TakeTripletComCam"]
+
+import abc
+import types
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.observatory.control.maintel.comcam import ComCam, ComCamUsages
+from lsst.ts.observatory.control.maintel.mtcs import MTCS
+
+
+class TakeTripletComCam(salobj.BaseScript, metaclass=abc.ABCMeta):
+    """Take triplet (intra-focal, extra-focal and in-focus images)
+    sequence with ComCam.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    * triplet {n} of {m}: before taking a triplet.
+
+    """
+
+    def __init__(self, index, descr="Take triplet with ComCam.") -> None:
+        super().__init__(index=index, descr=descr)
+
+        self.config = None
+        self.mtcs = None
+        self._comcam = None
+
+    @property
+    def camera(self) -> ComCam:
+        return self._comcam
```

Comment:
Is there really any need to do this? Why not just have `self.camera` as an attribute? Unless you are thinking about also having this script work for `LSSTCam`, in which case, you might want to do that already.